src/person.rb

```diff
@@ -3,20 +3,24 @@ class Person
   attr_accessor :name, :age
   attr_reader :id
 
-  def initialize(age, parent_permission: true, name: 'Unknown')
+  def initialize(age, name = 'Unknown', parent_permission: true)
     @id = rand(1.1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
   end
 
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+
+  def correct_name
+    @name
+  end
```

Comment:
Nice job implementing a correct name method to return the name attribute😉 But it is also a requirement that the person class inherits from the nameable😉.

<img width="771" alt="image" src="https://github.com/M-Anwar-Hussaini/School-Library/assets/64643995/7d8f78ee-96d9-42bd-8152-80de67cf8b8d">